lib/modules/dosomething/dosomething_campaign_group/dosomething_campaign_group.features.field_base.inc

```diff
@@ -221,48 +126,5 @@ function dosomething_campaign_group_field_default_field_bases() {
     'type' => 'text',
   );
 
-  // Exported field_base: 'field_signup_confirm_msg'.
-  $field_bases['field_signup_confirm_msg'] = array(
```

Comment:
This field is still referenced here: https://github.com/DoSomething/phoenix/blob/e4961813d27a19bca90cf9015bd5fb609e732cf1/lib/modules/dosomething/dosomething_signup/dosomething_signup.module#L576.
I think this `if` statement should be removed.